scalableminds/webknossos
# Move Frontend code out of app/assets to avoid recompile

### URL of deployed dev instance (used for testing):
- https://frontendoutofapp.webknossos.xyz

### Steps to test:
- should still start, compile, load, both in dev and production setup. also, work. (try tracing)
- scala changes should trigger only backend recompile, js changes should trigger only webpack recompile

### Issues:
- fixes #3732

------
